stdlib/build.rs

```diff
@@ -19,7 +19,6 @@ const DOC_DIR_PATH: &str = "./docs";
 
 /// Read and parse the contents from `./asm` into a `LibraryContents` struct, serializing it into
 /// `assets` folder under `std` namespace.
-#[cfg(not(feature = "docs-rs"))]
 fn main() -> Result<()> {
```

Comment:
I guess the idea here was to try fix #1208 - but seems like it wasn't really working.